### DISCUSSION TITLE
第3节课笔记(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#247] [笔记]
第3节课笔记(19班)

**Comment by hu-qi** [comment]
https://zhuanlan.zhihu.com/p/676753246

**Comment by Jiangzzzzl** [comment]
https://blog.csdn.net/2301_80618119/article/details/135466030?spm=1001.2014.3001.5501

**Comment by Sixdes** [comment]
https://bbs.csdn.net/topics/617906294?spm=1001.2014.3001.6377

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/03/note03.md

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou/article/details/135538269

**Comment by jts250** [comment]
https://zhuanlan.zhihu.com/p/677317801

**Comment by JankinTian** [comment]
https://blog.csdn.net/xiao_xian_/article/details/135586301

**Comment by romanwin28** [comment]
https://blog.csdn.net/George_han/article/details/135588324

**Comment by yunlong96** [comment]
https://blog.csdn.net/li_kin/article/details/135538535?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135538535%22%2C%22source%22%3A%22li_kin%22%7D

**Comment by mkdirmushroom** [comment]
[第3节课笔记](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial2_note.md)

**Comment by Hamasuta666** [comment]
第三节课笔记（作业在笔记末）：
https://zhuanlan.zhihu.com/p/680574294
